### ISSUE TITLE
Renaming Namespace

### ISSUE THREAD

**Issue by nasrulhazim** [apiato/apiato#189]
Hi there,

I've tried use the `php artisan app:name` to rename default `App` namespace. But afterwards, it seems that in the following files, the namespace did not get updated.

1. `apiato/bootstrap/app.php`

```
$app->singleton(
    Illuminate\Contracts\Http\Kernel::class,
    App\Ship\Engine\Kernels\ShipHttpKernel::class
);

$app->singleton(
    Illuminate\Contracts\Console\Kernel::class,
    App\Ship\Engine\Kernels\ShipConsoleKernel::class
);

$app->singleton(
    Illuminate\Contracts\Debug\ExceptionHandler::class,
    App\Ship\Engine\Exceptions\ShipExceptionsHandler::class
);
```

2. `apiato/config/app.php` - in `providers` key. 

```php
`App\Ship\Engine\Providers\PortoServiceProvider::class,`
```

**Comment by Mahmoudz** [comment]
@nasrulhazim this is not supported, not sure how important it is. But would be nice if added.
I do find and replace in my IDE for similar situations.
Best,

**Comment by nasrulhazim** [comment]
For me, I usually rename my application namespace when creating new application.

So, for me, it's quiet important. 

But as for now, I've manually rename the application namespace in those two files.

Still thinking how can I fix the issue. Hurm....

**Comment by nasrulhazim** [comment]
By default, Laravel update namespace in those two files. Tested just now. hurm..

**Comment by johannesschobel** [comment]
an `app:name` might break 3rd-party containers that are installed?
not sure about this, however.. Just a thought..

**Comment by nasrulhazim** [comment]
Found the reason it breaks - it's because of the namespace naming convention - see my [PR 190](https://github.com/apiato/apiato/pull/190). Waiting for PR approval.

**Comment by Mahmoudz** [comment]
closed by #190
